### PR TITLE
Make sender email configurable

### PR DESCRIPTION
**Pull request by bpedersen2**
Instead of hardcoding a cern sender make the sending email configurable.

### CODE REVIEW

**Comment by ThiefMaster** [comment]
I'm curious, how are you using the plugin? This is not one we expected others to use ;)

File: cronjobs_cern/indico_cronjobs_cern/plugin.py [hunk 1]
```diff
@@ -10,16 +10,21 @@
...
+from wtforms.fields import EmailField
+
 
 def _order_func(object_list):
     return sorted(object_list, key=lambda r: natural_sort_key(r[1].full_name))
 
 
 class SettingsForm(IndicoForm):
     _fieldsets = [
+        ('Sender', ['sender_email',]),
```
Comment (ThiefMaster) [inline cronjobs_cern/indico_cronjobs_cern/plugin.py:22]:
```suggestion
        ('Sender', ['sender_email']),    
```

only 1-tuples need this

File: cronjobs_cern/indico_cronjobs_cern/plugin.py [hunk 2]
```diff
@@ -33,6 +38,7 @@ class CERNCronjobsPlugin(IndicoPlugin):
     configurable = True
     settings_form = SettingsForm
     default_settings = {
+        'sender_email': '',
```
Comment (ThiefMaster) [inline cronjobs_cern/indico_cronjobs_cern/plugin.py:41]: since it's a CERN plugin i'd keep our default CERN address here instead of leaving it empty

**Comment by bpedersen2** [comment]
> I'm curious, how are you using the plugin? This is not one we expected others to use ;)

I try to use this to send regular reminders for a category of seminars ;)

File: cronjobs_cern/indico_cronjobs_cern/tasks.py [hunk 3]
```diff
@@ -45,7 +45,8 @@ def _get_category_events_query(start_dt, end_dt, category_ids):
 
 
 def _send_email(recipients, template):
-    email = make_email(from_address='noreply-indico-team@cern.ch', to_list=recipients, template=template, html=True)
+    from_address =  CERNCronjobsPlugin.settings.get('sender_mail')
```
Comment (ThiefMaster) [inline cronjobs_cern/indico_cronjobs_cern/tasks.py:48]:
```suggestion
    from_address = CERNCronjobsPlugin.settings.get('sender_mail')
```

**Comment by ThiefMaster** [comment]
@bpedersen2 Just waiting for the changes from my comments before merging this :)

File: cronjobs_cern/indico_cronjobs_cern/plugin.py [hunk 4]
```diff
@@ -10,16 +10,21 @@
...
     return sorted(object_list, key=lambda r: natural_sort_key(r[1].full_name))
 
 
 class SettingsForm(IndicoForm):
     _fieldsets = [
+        ('Sender', ['sender_email']),
         ('Seminar emails', ['seminar_categories', 'seminar_recipients'])
     ]
 
+    sender_email = EmailField ('Sender')
```
Comment (ThiefMaster) [inline cronjobs_cern/indico_cronjobs_cern/plugin.py:26]:
```suggestion
    sender_email = EmailField('Sender', [Email()])
```

(also import that validator)

Comment (bpedersen2) [reply cronjobs_cern/indico_cronjobs_cern/plugin.py]: Updated

**Pull request merged**